lib/cldr/thor.rb

```diff
@@ -2,37 +2,54 @@
 
 require "thor"
 require "cldr"
+require "cldr/download"
 
 module Cldr
   class Thor < ::Thor
     namespace "cldr"
 
-    desc "download [--version=34] [--target=./vendor] [--source=http://unicode.org/Public/cldr/34/core.zip]",
-      <<~DESCRIPTION
-        Download and extract CLDR data:
-          * Use the --version parameter to set the release version of the CLDR data to use
-          * Use the --target parameter to specify where on the filesystem to extract the downloaded data
-          * Use the --source parameter to override the location of the CLDR zip to download. Overrides --version
-      DESCRIPTION
-    method_options ["source", "-s"] => :string,
-      ["target", "-t"] => :string,
-      ["version", "-v"] => :string
-
+    desc "download [--version=#{Cldr::Download::DEFAULT_VERSION}] [--target=#{Cldr::Download::DEFAULT_TARGET}] [--source=#{format(Cldr::Download::DEFAULT_SOURCE, version: Cldr::Download::DEFAULT_VERSION)}]", "Download and extract CLDR data"
+    option :source, aliases: [:s], type: :string,
+      default: Cldr::Download::DEFAULT_SOURCE,
+      banner: Cldr::Download::DEFAULT_SOURCE,
+      desc: "Release version of the CLDR data to use"
+    option :target, aliases: [:t], type: :string,
+      default: Cldr::Download::DEFAULT_TARGET,
+      banner: Cldr::Download::DEFAULT_TARGET,
+      desc: "Where on the filesystem to extract the downloaded data"
+    option :version, aliases: [:v], type: :numeric,
+      default: Cldr::Download::DEFAULT_VERSION,
+      banner: Cldr::Download::DEFAULT_VERSION,
+      desc: "Override the location of the CLDR zip to download. Overrides --version"
```

Comment:
I don't understand the last part of this description. Did you intend for this to mean it overrides a different attribute, or are the descriptions for `source` and `version` swapped?

Reply:
> or are the descriptions for `source` and `version` swapped?

Good catch. Yes. Fixed.